### PR TITLE
Terraform resource module index updates

### PR DESCRIPTION
**Pull request by lonegunmanb**
New index for terraform-azurerm-avm-res-cognitiveservices-cognitive-account module

# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## As part of this Pull Request I have

- [ ] Read the Contribution Guide and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [ ] Ensured PR tests are passing
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by segraef — APPROVED** [review]
lgtm, waiting for @Azure/avm-core-team-technical-terraform

File: docs/static/module-indexes/TerraformResourceModules.csv [hunk 1]
```diff
@@ -8,6 +8,7 @@ Microsoft.Cdn,profiles,CDN Profile,avm-res-cdn-profile,New Module :new:,https://
...
 Microsoft.Compute,virtualMachineScaleSets,Virtual Machine Scale Set,avm-res-compute-virtualmachinescaleset,New Module :new:,https://github.com/Azure/terraform-azurerm-avm-res-compute-virtualmachinescaleset,https://registry.terraform.io/modules/Azure/avm-res-compute-virtualmachinescaleset/azurerm/latest,46d3xgtf.res.compute-virtualmachinescaleset,terrymandin,Terry Mandin,marcelkmfst,Marcel Keller,@Azure/avm-res-compute-virtualmachinescaleset-module-owners-tf,@Azure/avm-res-compute-virtualmachinescaleset-module-contributors-tf,AVM Resource Module for Virtual Machine Scale Set,,
+Microsoft.CognitiveServices,accounts,Cognitive Account,avm-res-cognitiveservices-cognitive-account,New Module :new:,https://github.com/Azure/terraform-azurerm-avm-res-cognitiveservices-cognitive-account,https://registry.terraform.io/modules/Azure/avm-res-cognitiveservices-cognitive-account/azurerm/latest,46d3xgtf.res.cognitiveservices-cognitive-account,lonegunmanb,Zijie He,lonegunmanb,Zijie He,@Azure/avm-res-cognitiveservices-cognitive-account-module-owners-tf,@Azure/avm-res-cognitiveservices-cognitive-account-module-contributors-tf,AVM Resource Module for Cognitive Account,,2024-01
```
Comment (mbilalamjad) [inline docs/static/module-indexes/TerraformResourceModules.csv:11]: The last value "2024-01" for column "FirstPublishedIn" is only provided when the module is finally published into the registry, recommend removing this value for now till such time the module gets published.

**Comment by matebarabas** [comment]
@mbilalamjad, @PmeshramPM, can you please help with this one? It doesn't seem to be added to the main index spreadsheet, and the name is wrong (it should be 'avm-res-cognitiveservices-account') - this mistake is propagated to all the other strings; please fix those too. Thanks!

**Comment by lonegunmanb** [comment]
> @mbilalamjad, @PmeshramPM, can you please help with this one? It doesn't seem to be added to the main index spreadsheet, and the name is wrong (it should be 'avm-res-cognitiveservices-account') - this mistake is propagated to all the other strings; please fix those too. Thanks!

I'll correct the name.

**Comment by lonegunmanb** [comment]
Hey @matebarabas @mbilalamjad I think I've updated this pr and repo's name, along with github team's names, would you please give this pr another review? Thanks!

**Comment by matebarabas** [comment]
@lonegunmanb, my apologies, after some additional corss-checks, I noticed that this seems to have been added by @PmeshramPM to the index, about 14 hours ago. See here: https://github.com/Azure/Azure-Verified-Modules/blob/47bf0eecdc61d0eb92ac989fa06f4ce908e250a8/docs/static/module-indexes/TerraformResourceModules.csv?plain=1#L46

Please feel free to abandon this PR.

**Comment by segraef** [comment]
Hi @lonegunmanb, solved via https://github.com/Azure/Azure-Verified-Modules/pull/580, this PR can be closed.

**Comment by jtracey93** [comment]
superseded by #614

**Pull request closed without merging**